packages/hardhat-plugin/test/calls.ts

```diff
@@ -1,27 +1,26 @@
 /* eslint-disable import/no-unused-modules */
-import { buildModule } from "@ignored/ignition-core";
+import { defineModule } from "@ignored/ignition-core";
 import { assert } from "chai";
 
-import { deployModule } from "./helpers";
-import { useEnvironment } from "./useEnvironment";
+import { useEphemeralIgnitionProject } from "./use-ignition-project";
 
-describe.skip("calls", () => {
-  useEnvironment("minimal");
+describe("calls", () => {
+  useEphemeralIgnitionProject("minimal-new-api");
```

Comment:
Open to better names! This adds a `this.deploy` in the same way that `useFileIgnitionProject` does for the nonce-checks.

This deploy is scoped to a fixture project but doesn't write to a deployments folder.